### PR TITLE
hco: add a SNO lane on Azure

### PR DESCRIPTION
**Pull request by tiraboschi**
hco: add a SNO lane on Azure

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

### CODE REVIEW

**Review by nunnatsa — APPROVED** [review]
/approve

File: ci-operator/step-registry/optional-operators/ci/azure/sno/optional-operators-ci-azure-sno-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"path": "optional-operators/ci/azure/sno/optional-operators-ci-azure-sno-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"dptp"
```
Comment (petr-muller) [inline ci-operator/step-registry/optional-operators/ci/azure/sno/optional-operators-ci-azure-sno-workflow.metadata.json:5]: DPTP does not want to own this

**Review by petr-muller — COMMENTED** [review]
We don't want to own anything else than the core OO workflows, otherwise this is fine

File: ci-operator/step-registry/optional-operators/ci/azure/sno/OWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+approvers:
+  - dptp
```
Comment (petr-muller) [inline ci-operator/step-registry/optional-operators/ci/azure/sno/OWNERS:2]: Please remove dptp from here, we do not want to own workflows.

**Pull request merged**